app/javascript/components/layout/CustomTable.js

```diff
@@ -138,62 +138,71 @@ class CustomTable extends React.Component {
             <Spinner variant="secondary" animation="border" size="lg" />
           </div>
         )}
-        <Table striped bordered hover size="sm">
-          <thead>
-            <tr>
-              {this.props.columnData.map(data => {
-                return this.renderTableHeader(data.field, data.label, data.isSortable, data.tooltip, data.icon, data.colWidth);
-              })}
-              {this.props.isEditable && <th>Edit</th>}
-              {this.props.isSelectable && (
-                <th>
-                  <input type="checkbox" onChange={this.toggleSelectAll} checked={this.props.selectAll}></input>
-                </th>
-              )}
-            </tr>
-          </thead>
-          <tbody>
-            {this.props.rowData.map((data, row) => {
-              return (
-                <tr key={row} id={data.id ? data.id : row}>
-                  {Object.values(this.props.columnData).map((col, index) => {
-                    let value = data[col.field];
-                    if (col.options) {
-                      // If this column has value options, use the data value as a key to those options
-                      value = col.options[data[col.field]];
-                    } else if (col.filter) {
-                      // If this column has a filter, apply the filter to the value
-                      // Send along string of the ID and HoH bool if needed
-                      value = col.filter(data[col.field], data.id?.toString(), data.is_hoh);
-                    }
-                    return (
-                      <td key={index} className={col.className ? col.className : ''}>
-                        {col.onClick && <span onClick={() => (col.onClick(data.id.toString()) ? col.onClick : null)}>{value}</span>}
-                        {!col.onClick && value}
+        <div></div>
+        <div className={this.props.getCustomTableClassName ? `table-responsive ${this.props.getCustomTableClassName()}` : 'table-responsive'}>
```

Comment:
FYI: This diff is large mostly because I added a div around the Table element for responsiveness and custom classes.

Other changes included that are now not so obvious (thanks GitHub):

- Updating variable names in the row/col loops.
- Passing more data to the provided col filter method (if any).
- New prop for row className.

-